styles/wikiwand/catppuccin.user.css

```diff
@@ -72,6 +101,7 @@
     --color-bg: @base;
     --color-text: @text;
     --color-grey: @subtext0;
+    --color-subtle: @subtext0;
```

Comment:
What does this theme?

Reply:
themes this
![image](https://github.com/user-attachments/assets/8fe4f8cb-447a-4abd-9ee1-47a61c5873f7)

Reply:
Mind sending me the link you found it on? Just want to see how it looks in context!

Reply:
sure, https://www.wikiwand.com/en/%22Hello,_World!%22_program